### PR TITLE
108 Update README to redirect installation process to the wiki.

### PR DESCRIPTION
**Pull request by thomas-senechal**
- Removes "How to redirect massa Top Level Domain to localhost ?"
- Adds "How to install Thyra on my computer ?"

### CODE REVIEW

**Comment by thomas-senechal** [comment]
@gregLibert I updated everything as asked. Tell me if there is any issue.

**Review by Thykof — APPROVED** [review]

**Review by Trottoir — APPROVED** [review]

**Review by Thykof — APPROVED** [review]

**Comment by SlnPons** [comment]
@gregLibert following your installation process, I cannot access Flappy. 
Also, could you update the link to start "your massalien journey" by: http://my.massa/thyra/wallet please? (in each OS section)

**Comment by Trottoir** [comment]
@SlnPons 
> @gregLibert following your installation process, I cannot access Flappy. Also, could you update the link to start "your massalien journey" by: http://my.massa/thyra/wallet please? (in each OS section)

What about deploying it trhough the UI? It may be not deployed on this DNS

File: INSTALLATION.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+echo 'address=/.massa/127.0.0.1' >> $(brew --prefix)/etc/dnsmasq.d/massa.conf
+sudo mkdir -p /etc/resolver
+sudo bash -c 'echo "nameserver 127.0.0.1" >> /etc/resolver/massa'
+
+echo "Now we need you to type in your password to start the dnsmasq service."
+sudo brew services start dnsmasq
+```
+
+Congratulations, you can now browse the **websites on-chain** seamlessly. If you need to take the pressure off, maybe a little [game](http://flappy.massa) can help.
+If you want to get down to business, you can start your [Massalian journey](http://my.massa) right away!
```
Comment (SlnPons) [inline INSTALLATION.md:77]:
```suggestion
If you want to get down to business, you can start your [Massalian journey](http://my.massa/thyra/wallet) right away!
```

File: INSTALLATION.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+1. Open Acrylic config file: Open Acrylic DNS Proxy UI > File > Open Acrylic Hosts
+
+2. Add `*.massa` top level domain to `AcrylicHosts.txt`: 
+```txt
+127.0.0.1   *.massa
+```
+3. Save the file and reload Acrylic: Open Acrylic DNS Proxy UI > Actions > Restart Acrylic Service
+
+Congratulations, you can now browse the **websites on-chain** seamlessly. If you need to take the pressure off, maybe a little [game](http://flappy.massa) can help.
+If you want to get down to business, you can start your [Massalian journey](http://my.massa) right away!
```
Comment (SlnPons) [inline INSTALLATION.md:153]: If you want to get down to business, you can start your [Massalian journey](http://my.massa/thyra/wallet) right away!

File: README.md [hunk 3]
```diff
@@ -65,37 +72,6 @@ Prerequisite: Having a running thyra-server application on your machine.
 To access the website you need to go to http://localhost:8080/website?url=<address of the website>.
 For instance, to access flappy text stored on the blockchain, click the following link: http://localhost:8080/website?url=A1aMywGBgBywiL6WcbKR4ugxoBtdP9P3waBVi5e713uvj7F1DJL.
```
Comment (Trottoir) [inline README.md:73]: Need to change that contract address with the right flappy address

Comment (Trottoir) [reply README.md:73]: FYI ATM we don't have a way to deploy automatically a website on a choosen address

File: INSTALLATION.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+sudo mkdir -p /etc/NetworkManager/dnsmasq.d/
+echo "address=/.massa/127.0.0.1" >> /etc/NetworkManager/dnsmasq.d/massa.conf
+sudo mv /etc/resolv.conf /etc/resolv.conf_backup_thyra_install && sudo ln -s /var/run/NetworkManager/resolv.conf /etc/resolv.conf
+sudo systemctl restart NetworkManager
+```
+
+> **_NOTE:_** your `/etc/resolv.conf` file has been backed up to `/etc/resolv.conf_backup_thyra_install`.
+
+Congratulations, you can now browse the **websites on-chain** seamlessly. If you need to take the pressure off, maybe a little [game](http://flappy.massa) can help.
+If you want to get down to business, you can start your [Massalian journey](http://my.massa) right away!
```
Comment (SlnPons) [inline INSTALLATION.md:123]:
```suggestion
If you want to get down to business, you can start your [Massalian journey](http://my.massa/thyra/wallet) right away!
```

File: INSTALLATION.md (hunk 2 above)
Comment (Thykof) [inline INSTALLATION.md:153]:
```suggestion
If you want to get down to business, you can start your [Massalian journey](http://my.massa/thyra/wallet) right away!
```

**Review by Thykof — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Thykof) [reply README.md:73]: you are tight, so this address would work only in the right massa network. Should we specify the network this URL is supposed to work?

**Review by Thykof — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (Thykof) [inline README.md:73]:
```suggestion
```

Comment (Thykof) [reply README.md:73]: see https://github.com/massalabs/thyra/pull/114/files#r984576641

**Review by Thykof — CHANGES_REQUESTED** [review]

File: INSTALLATION.md (hunk 4 above)
Comment (Thykof) [reply INSTALLATION.md]: done

File: README.md [hunk 5]
```diff
@@ -65,37 +72,6 @@ Prerequisite: Having a running thyra-server application on your machine.
 To access the website you need to go to http://localhost:8080/website?url=<address of the website>.
```
Comment (Thykof) [inline README.md:72]:
```suggestion
To access the website you need to go to http://localhost:80/website?url=<address of the website>.
```

**Pull request merged**